go/obscuronode/enclave/enclaverunner/cli_flags.go

```diff
@@ -46,4 +46,7 @@ const (
 
 	ViewingKeysEnabledName  = "viewingKeysEnabled"
 	ViewingKeysEnabledUsage = "Whether the enclave should encrypt responses to sensitive requests with viewing keys"
+
+	edgelessDBHostName  = "edgelessDBHost"
+	edgelessDBHostUsage = "Whether the enclave will use an in-memory DB rather than persist data"
```

Comment:
@BedrockSquirrel This is confusing. Above, in `enclave_config.go`, it says this is the address for the DB. But here, it sounds like a boolean flag? Can you align the descriptions, please?